static/src/javascripts/projects/common/modules/commercial/acquisitions-link-enrichment.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import { addReferrerData } from 'common/modules/commercial/acquisitions-ophan';
+import fastdom from 'lib/fastdom-promise';
+
+const addReferrerDataToAcquisitionLink = (rawUrl: string): string => {
+    const acquisitionDataField = 'acquisitionData';
+
+    let url;
+    try {
+        url = new URL(rawUrl);
+    } catch (e) {
+        return rawUrl;
+    }
+
+    let acquisitionData;
+    try {
+        acquisitionData = JSON.parse(
+            url.searchParams.get(acquisitionDataField)
+        );
+    } catch (e) {
+        return rawUrl;
+    }
+
+    if (acquisitionData) {
+        acquisitionData = addReferrerData(acquisitionData);
+        url.searchParams.set(
+            acquisitionDataField,
+            JSON.stringify(acquisitionData)
+        );
+    }
+
+    return url.toString();
+};
+
+const ACQUISITION_LINK_CLASS = 'js-acquisition-link';
+
+const addReferrerDataToAcquisitionLinksOnPage = (): void => {
+    const links = Array.from(
+        document.getElementsByClassName(ACQUISITION_LINK_CLASS)
+    );
```

Comment:
To follow @joelochlann's suggestion, we could make this more concise using [array destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment)
```js
const links = [...document.getElementsByClassName(ACQUISITION_LINK_CLASS)];
```